packages/amplify-cli/bin/amplify

```diff
@@ -4,6 +4,10 @@
 // this file should be a duplicate of amplify.production.template except for the CLI_ENV value
 process.env.CLI_ENV = 'development';
 
+// suppress deprecation warnings
```

Comment:
Shall we keep this as false for development ?

Reply:
Yes. However, we'll get this back after we deal with cloudform.

Reply:
https://app.asana.com/0/1204191501160861/1204261257618498/f